add-ons/service.iptv.manager
# Improve tests and run tests on Windows

Tests:
* Replace own stubs with `sakee`. Requires https://github.com/retrospect-addon/kodi.emulator.ascii/pull/25 to be merged.
* Replace `coverage` with `pytest-cov`
* Remove `tox`
* Run service with pylint instead of starting it directly
* Run CI on Windows (Python 2.7, Python 3.8 and Python 3.9)

Other:
* Remove unused methods in `kodiutils`

Github Actions:
* Don't run actions twice (push & pull_request), see https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662
* Use `xbmc/action-kodi-addon-checker@v1.2` action instead of installing ourself
* Use output of `xbmc/action-kodi-addon-submitter` for the Github release
